source/Willow-Core-Tests/ISODateCodecTest.class.st

```diff
@@ -28,7 +28,8 @@ ISODateCodecTest >> testCantDecodeDate [
 		should: [ ISODateCodec new decode: '010-13-11' ] raise: InstanceCreationFailed;
 		should: [ ISODateCodec new decode: '2010-13-11' ] raise: InstanceCreationFailed;
 		should: [ ISODateCodec new decode: '2010-1-11' ] raise: InstanceCreationFailed;
-		should: [ ISODateCodec new decode: '2010-13-1' ] raise: InstanceCreationFailed
+		should: [ ISODateCodec new decode: '2010-13-1' ] raise: InstanceCreationFailed;
+		should: [ ISODateCodec new decode: '2020/12/11' ] raise: InstanceCreationFailed
 ]
```

Comment:
`#should:raise: ` must not be used unless there is no other alternative, since it can confuse different errors.
All references encountered during development and the new assertions should be changed to:
`#should:raise:withMessageText:`